zero/src/prover/cli.rs

```diff
@@ -47,6 +47,9 @@ pub struct CliProverConfig {
     /// be greater than zero.
     #[arg(long, default_value_t = 16)]
     block_pool_size: usize,
+    /// If true, returns the struct_logs along with the generation data.
+    #[arg(long, help_heading = HELP_HEADING, default_value_t = false)]
+    get_struct_logs: bool,
```

Comment:
I think @einar-polygon has [alternative cli option](https://github.com/0xPolygonZero/zk_evm/pull/427/files#diff-ddb94e68bba47a242773d5c3d673775d5694fb86b02a1d6f66a62440321ac667)  which will conflict with this one.

Reply:
Yes, Einar's PR definitely conflicts with this one, but I wanted to open the PR against `develop` since the changes are unrelated to Einar's. I will rebase when his is ready, but I think merging now with his PR might create a lot of "noise" for reviewers, don't you think?

Reply:
Maybe you could make the PR against his branch, then merge to `develop` after his PR is included there?